### DISCUSSION TITLE
Galaxy Badge 2

### DISCUSSION THREAD

**Discussion by severdia** [Schweinepriester/achievement-testarea#30] [Q&A]
Is this even possible? @4lfant

**Comment by xommmax** [comment]


**Reply by severdia** [reply]
This is it!

**Reply by xommmax** [reply]
@severdia could u please this too?

**Reply by xommmax** [reply]
@severdia yeah man, it worked. Awesome!
Have a great day!
would also be glad if u star my repo https://github.com/4lfant/flutter-guidelines 🥰

**Reply by severdia** [reply]
You too! Done. :)

**Reply by hoseinpur** [reply]
I wanna test this too! :(

**Comment by pulkitxm** [comment]
best Answer
